test/test_t8code_3d.jl

```diff
@@ -336,6 +352,39 @@ mkdir(outdir)
         end
     end
 
+    # This test is identical to the one in `test_p4est_3d.jl` besides minor
+    # deviations in the expected error norms.
```

Comment:
```suggestion
    # This test is identical to the one in `test_p4est_3d.jl`
```
I just checked: The error norms are _exactly_ equal aren't they, cf. https://github.com/trixi-framework/Trixi.jl/blob/1f31fd47c6c6faf9cba1ab90c6ec7569209e7865/test/test_p4est_3d.jl#L492-L504.

Reply:
Thanks again!
Yes, they should be identical. At some point they were not, but eventually @jmark fixed it.